generated/nidaqmx/_grpc_time.py

```diff
@@ -14,7 +14,7 @@
 _YS_PER_FS = 10**9
 
 
-def convert_time_to_timestamp(dt: Union[std_datetime, ht_datetime], ts: GrpcTimestamp) -> None:
+def convert_time_to_timestamp(dt: Union[std_datetime, ht_datetime], ts: Optional[GrpcTimestamp] = GrpcTimestamp()) -> GrpcTimestamp:
```

Comment:
Using mutable default arguments like this is a very common Python pitfall.
https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument

The correct way to do this is to put `ts: Optional[GrpcTimestamp] = None` in the function signature and put this in the function body:
```
if ts is None:
    ts = GrpcTimestamp()
```

BTW, `typing.Optional[T]` means "this parameter is either T or None". This is entirely orthogonal to optional parameters or default arguments. You can have a required parameter that uses `typing.Optional` like `x: Optional[str]`, or you can have an optional parameter that doesn't use `typing.Optional` like `x: str = ""`.